server/src/main/java/org/elasticsearch/index/mapper/MapperFeatures.java

```diff
@@ -46,7 +46,8 @@ public Set<NodeFeature> getFeatures() {
             IndexSettings.IGNORE_ABOVE_INDEX_LEVEL_SETTING,
             SourceFieldMapper.SYNTHETIC_SOURCE_COPY_TO_INSIDE_OBJECTS_FIX,
             TimeSeriesRoutingHashFieldMapper.TS_ROUTING_HASH_FIELD_PARSES_BYTES_REF,
-            FlattenedFieldMapper.IGNORE_ABOVE_WITH_ARRAYS_SUPPORT
+            FlattenedFieldMapper.IGNORE_ABOVE_WITH_ARRAYS_SUPPORT,
+            RangeFieldMapper.DATE_RANGE_INDEXING_FIX
```

Comment:
This can be a test feature I think...

Reply:
Wasn't sure about when to use test featuers or not; thanks for the suggestion.  Let me know if that's not what you were thinking.